runtime/near-runtime-fees/src/lib.rs

```diff
@@ -181,6 +181,7 @@ impl Default for RuntimeFeesConfig {
                 },
             },
             storage_usage_config: StorageUsageConfig {
+                // amount - 16, locked - 16, code_hash - 32, storage_used - 4, account_id hash - 32
```

Comment:
It's all random here. Not sure what it suppose to represent. Maybe it was:
`amount - 16, locked - 16, code_hash - 32, storage_used - 8, storage_paid_by - 8`.
It's total of 80. Not sure what's the remaining 20 was for. There is no `account_id hash` yet, so it shouldn't be in the comment.
@mikhailOK

Reply:
Let's remove the comment for now

Reply:
I thought we already switched to use account hash as trie root? Let's count it here like this for now - until we figure out the correct storage amount required to charge. E.g. we don't charge for account id anywhere else.